lib/seahorn/BvOpSem2ExtraWideMemMgr.cc

```diff
@@ -407,12 +407,11 @@ ExtraWideMemManager<T>::ptrAdd(ExtraWideMemManager::PtrTy base,
   // do concrete computation if possible
   if (m_ctx.alu().isNum(base.getOffset())) {
     signed conc_offset = m_ctx.alu().toNum(base.getSize()).get_si() + _offset;
-    new_offset = m_ctx.alu().si(conc_offset, ptrSizeInBits());
+    new_offset = m_ctx.alu().ui(conc_offset, ptrSizeInBits());
```

Comment:
why is this `ui` and not `si`, since we allow negative offset in symbolic computation in line 412

Reply:
goot catch. will be fixed in next revision.